### PR TITLE
Refactor wizard configuration code

### PR DESCRIPTION
**Pull request by irlaec**

### CODE REVIEW

File: octoprint_mrbeam/wizard_config.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+class WizardConfig:
+	def __init__(self, plugin):
+		self._logger = mrb_logger("octoprint.plugins.mrbeam.wizard_config")
+
+		self._plugin = plugin
+		self._user_manager = plugin._user_manager
+		self._plugin_manager = plugin._plugin_manager
+		self._settings = plugin._settings
+
+		self._wizard_version = 16  # random number. but we can't go down anymore, just up.
```
Comment (andywerner) [inline octoprint_mrbeam/wizard_config.py]: This should be a constant

File: octoprint_mrbeam/wizard_config.py [hunk 2]
```diff
@@ -0,0 +1,149 @@
+from flask.ext.babel import gettext
+from octoprint_mrbeam.mrb_logger import mrb_logger
+
+
+class WizardConfig:
+	def __init__(self, plugin):
```
Comment (andywerner) [inline octoprint_mrbeam/wizard_config.py:6]: Could you please add a comment that explains where the order of the wizard-tabs is defined. Just to help our future selfs. thx

File: octoprint_mrbeam/wizard_config.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+				mandatory=False,
+				suffix='_analytics',
+				template='wizard/wizard_analytics.jinja2',
+				div='wizard_plugin_corewizard_analytics',
+			)
+		)
+
+		return welcome_wizard_tabs
+
+	@staticmethod
```
Comment (andywerner) [inline octoprint_mrbeam/wizard_config.py]: Why is this one a static method while _welcome_wizard_config() is a regular member?
Does the call to this one even work with self._whatsnew_wizard_config() (line 30)

**Review by andywerner — CHANGES_REQUESTED** [review]

File: octoprint_mrbeam/wizard_config.py (hunk 2 above)
Comment (irlaec) [reply octoprint_mrbeam/wizard_config.py:6]: Good idea

File: octoprint_mrbeam/wizard_config.py (hunk 3 above)
Comment (irlaec) [reply octoprint_mrbeam/wizard_config.py]: Changed

File: octoprint_mrbeam/wizard_config.py (hunk 1 above)
Comment (irlaec) [reply octoprint_mrbeam/wizard_config.py]: Done

**Pull request merged**